ansible/roles/satellite-manage-manifest/tasks/version_6.x.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Check if manifest file is uploaded
+  stat:
+    path: "/tmp/{{ manifest_file | basename }}"
+  register: manifest_file_existance_result
+
+- name: Copy manifest
+  copy:
+    src: "{{ manifest_file }}"
+    dest: /tmp
+  when:
+    - not manifest_file_existance_result.exists
+    - "not manifest_file.startswith('http')"
+    - "not manifest_file.startswith('ftp')"
```

Comment:
Very nasty check whether the file is local or not :thinking:
Any suggestions how to improve it? :upside_down_face:

Reply:
That test is not necessary.

ansible module `copy` will not copy if the files already exist and is the same checksum or content.